lib/calendar.ts

```diff
@@ -1691,8 +1691,11 @@ const makeHelperGregorian = (id: BuiltinCalendarId, originalEras: InputEra[]) =>
     eras,
     anchorEra,
     calendarType: 'solar',
-    inLeapYear(calendarDate /*, cache */) {
-      const { year } = this.estimateIsoDate(calendarDate);
+    inLeapYear(calendarDate /*, cache: OneObjectCache */) {
```

Comment:
Do any of the overriding calendars use the month and day to determine `inLeapYear`? If not, maybe we could just pass `year: number` to this method instead of what is effectively `calendarDate: { year: number }`?